README.md

```diff
@@ -113,8 +136,8 @@ You can deploy this project using:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **Deploy on GitHub**
 - [ ] **Responsive design**
+- [ ] **Stock Market Analysis page**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] In your readme acknowledgment section kindly add a credit to Nelson Sakwa whose design was used for the project.